src/commands/MemoryAddCommand.js

```diff
@@ -0,0 +1,20 @@
+import AbstractCommand from './AbstractCommand';
+
+export default class MemoryAddCommand extends AbstractCommand {
+    execute() {
+        if (localStorage.getItem('memoryValue')) {
+            const prevStr = localStorage.getItem('memoryValue');
+            const cur = +this.firstValue + +prevStr;
```

Comment:
```suggestion
            const cur = parseInt(this.firstValue, 10) + parseInt(prevStr, 10);
```

Приводи к числу, чтобы было видно явно